README.md

```diff
@@ -1,18 +1,17 @@
 Portfolio to Microverse course
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio Microverse
+# Awesome Book-List Microverse
 
-> Capstone Project-1 using HTML, CSS, JavaScript
+>Awesome Book-List using HTML, CSS, JavaScript
 
-### Check out in this [link](https://rebel216.github.io/Capstone-Project-1/)
+### Check out in this [link]()
 
 
 
 ## Acknowledgements
 
-Credit to Cindy Shin, the author of the original design, `Creative Commons License`.
-Original design idea by Cindy Shin in Behance.
+
```

Comment:
- [ ] Kindly add the correct description, link  and acknowledgment to your readme , this will ensure your project is appropriately documented 👍🏿

Reply:
done